templates/repo/settings/protected_branch.tmpl

```diff
@@ -5,38 +5,48 @@
 				{{.locale.Tr "repo.settings.branch_protection" (.Rule.RuleName|Escape) | Str2html}}
 			</h4>
 			<div class="ui attached segment branch-protection">
+				<h5 class="ui dividing header">{{.locale.Tr "repo.settings.protect_patterns"}}</h5>
```

Comment:
Yes, this is a kind of [variation for header](https://fomantic-ui.com/elements/header.html#dividing), looks like `.gt-border-secondary-bottom` has similar effect, I am ok with both.

Reply:
My preference is: if the framework doesn't tell me to do so, and it looks counterintuitive (why a "divider" could have "content"?), then I won't do that. 

Otherwise something would break in the future.

Reply:
Actually this is not a divider but a header, and it is already used in many places, like on new project page, with style defined in `base.css`

<img width="825" alt="Screen Shot 2023-04-27 at 16 14 25" src="https://user-images.githubusercontent.com/17645053/234801696-fcfd1c93-124b-4901-af61-b5d0bb69f08e.png">

<img width="412" alt="Screen Shot 2023-04-27 at 16 16 22" src="https://user-images.githubusercontent.com/17645053/234802557-b6fcbc64-3ba7-4423-8620-e4f7cf87b88b.png">

I can change this to `.gt-border-secondary-bottom` anyways as I have no preference.

Reply:
> Actually this is not a divider but a header, and it is already used in many places,

Oh I see, it's "dividing" not "divider", so L-G-T-M.